recipes/yt/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Also, could you please add `--single-version-externally-managed --record=record.txt` here, as well?